### PR TITLE
Add test for public quay image

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: src/main/resources/standardTestCases/docker_hash/docker_hash_quay.wdl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+task quay {
+    command {
+        echo "hello"
+    }
+    runtime {
+        docker: "quay.io/tjeandet/tj-ubuntu:centaur"
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/docker_hash/docker_hash_quay.wdl]: Is there a more official ubuntu version?

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/182/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/182/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/docker_hash/docker_hash_quay.wdl (hunk 1 above)
Comment (Horneth) [reply src/main/resources/standardTestCases/docker_hash/docker_hash_quay.wdl]: I can't find an "official repositories" equivalent of dockerhub for quay but http://www.appcontainers.com/ have an ubuntu repo on quay so I can use that instead

**Pull request merged**